int64 mode
---
bool success
string message
